app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -210,8 +209,14 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         }
 
         _webView.setBackgroundColor(Color.TRANSPARENT);
+
         final Bundle args = getArguments();
-        setViewModeVisibility(args.getBoolean(START_PREVIEW, _appSettings.getDocumentPreviewState(_document.getPath())));
+        final boolean startInPreview = _appSettings.getDocumentPreviewState(_document.getPath());
+        if (args != null && savedInstanceState == null) { // Use the launch flag on first launch
+            setViewModeVisibility(args.getBoolean(START_PREVIEW, startInPreview));
```

Comment:
ok so looks like you added something here already on the quick, nice! thanks! I wanted to do the same now, but so don't 😄